gc/base/GCExtensionsBase.hpp

```diff
@@ -1513,8 +1523,10 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		, isArrayletDoubleMapRequested(false)
 		, isArrayletDoubleMapAvailable(false)
 #endif /* defined(OMR_GC_DOUBLE_MAP_ARRAYLETS) */
-		, isVirtualLargeObjectHeapRequested(false)
+#if defined(OMR_GC_SPARSE_HEAP_ALLOCATION)
+		, isVirtualLargeObjectHeapRequested(true)
```

Comment:
we will probably enable off-heap by default once all the code is delivered, but it should be done in downstream project during startup